client/src/App.css

```diff
@@ -44,4 +44,10 @@
 
 .read-the-docs {
   color: #888;
-}
\ No newline at end of file
+}
+
+ul.li {
```

Comment:
please, be more specific 
someone else could have <ul> and <li> elements in other components.
decorate these elements with some additional class names or ids